tests/caasp/oci_install.pm

```diff
@@ -8,15 +8,26 @@
 # without any warranty.
 
 # Summary: Start CaaSP installation
-# Maintainer: Martin Kravec <mkravec@suse.com>
+# Maintainer: Martin Kravec <mkravec@suse.com>, Panagiotis Georgiadis <pgeorgiadis@suse.com>
 
 use strict;
 use warnings;
 use base "y2logsstep";
+#use utils qw(handle_login handle_emergency get_x11_console_tty);
```

Comment:
No disabled code please

Reply:
Actually none of those are used even before this patch.I will remove the line. I should have done this in the first place but I forgot about it :/ thanks for catching this!